modules/assisted-installer-setting-the-cluster-details.adoc

```diff
@@ -0,0 +1,42 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: PROCEDURE
+[id='setting-the-cluster-details_{context}']
+
+= Setting the cluster details
+
+To create a cluster with the {ai-full} web user interface, use the following procedure.
+
+.Procedure
+
+. Log in to the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[RedHat Hybrid Cloud Console].
+
+. Open the menu on the top left and click *OpenShift*.
+
+. Click *Create cluster*.
+
+. Click the *Datacenter* tab.
+
+. Under the *{ai-full}* section, select *Create cluster*.
+
+. Enter a name for the cluster in the *Cluster name* field.
+
+. Enter a base domain for the cluster in the *Base domain* field. All subdomains for the cluster will use this base domain.
++
+[NOTE]
+====
+The base domain must be a valid DNS name. You must not have a wild card domain set up for the base domain. You cannot change the base domain after cluster installation.
```

Comment:
I'd also drop a note somewhere about some of those fields not being able to get updated after cluster creation, e.g.: sno-checkbox, CPU architecture, network configuration (dynamic/static) and disk-encryption 
![Screenshot from 2022-08-30 10-36-45](https://user-images.githubusercontent.com/7492909/187378050-663bc899-39dc-49cb-abfa-dbedf6384c3c.png)